modules/bending_energy_diffusion_loss_notes.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>Here is a way I would describe the conclusion: We saw above that unnormalized diffusion loss is sensitive to image <em>aspect ratio</em> but not overall image scale. On the other hand we see here that unnormalized bending energy loss <em>is sensitive to overall image scale</em>, making normalization even more important for bending energy loss.</p><p> </p><p>Then here is a suggestion for the table to get the idea across:</p><p> </p><p>|                    | image aspect ratio(s)  | overall image scale | </p><p>| :----------------: | :-------------: | :-------------: | </p><p>| (unnormalized) <strong>Bending energy</strong> | Variant         | Variant         | </p><p>| (unnormalized) <strong>Diffusion</strong>      | Invariant       | Variant         | </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
(just realized my columns are backwards and edited to fix)